e2e/cypress/support/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+declare namespace Cypress {
+
+    // We cannot use normal imports; otherwise, we would convert this to a
+    // normal module, and we want it to be an ambient module in order to merge
+    // our declaration of Cypress with the global one.
+    // See https://stackoverflow.com/a/51114250/3248221 for more information
+    type Channel = import('mattermost-redux/types/channels').Channel;
+    type Team = import('mattermost-redux/types/teams').Team;
+    type UserProfile = import('mattermost-redux/types/users').UserProfile;
+    type AdminConfig = import('mattermost-redux/types/config').AdminConfig;
+    type PreferenceType = import('mattermost-redux/types/preferences').PreferenceType;
+
+    type PartialAdminConfig = import('./server_api_commands').PartialAdminConfig;
+    type MessageDisplay = import('./server_api_commands').MessageDisplay;
+    type TeammateNameFormat = import('./server_api_commands').TeammateNameFormat;
```

Comment:
3 types above were not used elsewhere and can't be found in the stated path.

Reply:
done.